AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Sync/RemoteSyncEngineTests.swift

```diff
@@ -57,6 +59,7 @@ class RemoteSyncEngineTests: XCTestCase {
         remoteSyncEngine.start()
 
         wait(for: [failureOnStorageAdapter], timeout: defaultAsyncWaitTimeout)
+        remoteSyncEngineSink.cancel()
```

Comment:
Just curious, did you run into some problem with remote sync engine needing these cancels?

Reply:
No, it's just there to get rid of the "unused variable" compiler warning. (We can't do away with the variable, or else Combine releases the sink before it has a chance to process.)